content/docs/user-guide/assets/pipeline/metadata.md

```diff
@@ -0,0 +1,85 @@
+        "AssetProcessor":
+        {
+            "Settings":
+            {
+                "Metadata":
+                {
+                    "EnabledTypes": [
+                    ]
+                }
+            }
+        }
+    }
+}
+```
+EnabledTypes is simply an array of strings of file extension to enable.  For example, `".txt", ".png"`
+
+It is highly recommened that this setreg file be placed in a central location which is shared with other users of the project to ensure everyone is generating the same metadata files.  For a game project, it should be placed in the project directory; for gem development, it should be placed in the gem directory.  Keep in mind this is an Asset Processor wide setting, it will apply to all files the AP processes, not just the ones in the project/gem where the setting exists.
+
+Once a metadata file is generated for an asset and the asset is relocated or any references are saved using the newly generated UUID, the metadata file must be kept alongside the asset.  Not doing so could break references to the file.
+
+Note that AP will continue to use existing metadata files for assets which have them regardless of the above settings.
+
+## Renaming while AP is running
+By default, renaming assets manually while Asset Processor is running can prove difficult as AP will try to immediately create a new metadata file for the renamed asset.  To prevent this, AP can be configured to wait for a specified time before it attempts to process a "new" asset, which avoids the automatic creation of an unwanted, new metadata file.  Note that this delay will affect all newly created assets for metadata-enabled types, which is why this is off by default.  The longer the delay time, the longer it will take AP to start processing any newly created metadata-enabled file type.  In a .setreg file, add the following setting:
+```json
+{
+    "O3DE":
+    {
+        "AssetProcessor":
+        {
+            "Settings":
+            {
+                "Metadata":
+                {
+                    "MetaCreationDelayMs": 5000
+                }
+            }
+        }
+    }
+}
+```
+The value specified is the milliseconds to wait.  The above example config will wait 5 seconds before processing an asset.  Note that this setting can be user-specific and does not need to be shared across users if not desired.
+
+This setting is mainly useful to help allow manual renaming of assets, but can help in some situations of slow file updates on disk, such as copying a large number of files or fetching latest from source control.  These operations can sometimes be ordered in a way that requires a very high delay setting, such as a Git LFS fetch which may fetch files in groups by type, possibly resulting in metadata files being created on disk long after the source asset was created.  In these situations, it is recommended to close AP first to avoid conflicts caused by AP creating metadata files which already exist in source control.

```

Comment:
```suggestion
This setting is mainly useful to help allow manual renaming of assets, but can help in some situations of slow file updates on disk, such as copying a large number of files or fetching latest from source control.  These operations can sometimes be ordered in a way that requires a very high delay setting, such as a Git LFS fetch, which may fetch files in groups by type, possibly resulting in metadata files being created on disk long after the source asset was created.  In these situations, it is recommended to close AP first to avoid conflicts caused by AP creating metadata files which already exist in source control.

```